custom_components/bermuda/coordinator.py

```diff
@@ -136,7 +138,11 @@ def __init__(
 
         self.stamp_last_update: float = 0  # Last time we ran an update, from MONOTONIC_TIME()
         self.stamp_last_prune: float = 0  # When we last pruned device list
+        file_path = Path(__file__).parent / "manufacturer_identification" / "member_uuids.yaml"
 
+        with file_path.open("r") as f:
+            member_uuids_yaml = yaml.safe_load(f)["uuids"]
+        self.member_uuids = {hex(member["uuid"])[2:]: member["name"] for member in member_uuids_yaml}
```

Comment:
I get a warning about blocking IO being done outside of the executor loop.
https://developers.home-assistant.io/docs/asyncio_blocking_operations/#open

This seems to work:

```suggestion

        self.member_uuids = {}
        def load_manufacturer_ids():
            """Import yaml file containing manufacturer name mappings."""
            file_path = Path(__file__).parent / "manufacturer_identification" / "member_uuids.yaml"

            with file_path.open("r") as f:
                member_uuids_yaml = yaml.safe_load(f)["uuids"]
            self.member_uuids = {hex(member["uuid"])[2:]: member["name"] for member in member_uuids_yaml}

        hass.async_add_executor_job(load_manufacturer_ids)

```

Reply:
> I get a warning about blocking IO being done outside of the executor loop.
> 
> https://developers.home-assistant.io/docs/asyncio_blocking_operations/#open
> 
> 
> 
> This seems to work:
> 
> 
> 
> ```suggestion
> 
> 
> 
>         self.member_uuids = {}
> 
>         def load_manufacturer_ids():
> 
>             """Import yaml file containing manufacturer name mappings."""
> 
>             file_path = Path(__file__).parent / "manufacturer_identification" / "member_uuids.yaml"
> 
> 
> 
>             with file_path.open("r") as f:
> 
>                 member_uuids_yaml = yaml.safe_load(f)["uuids"]
> 
>             self.member_uuids = {hex(member["uuid"])[2:]: member["name"] for member in member_uuids_yaml}
> 
> 
> 
>         hass.async_add_executor_job(load_manufacturer_ids)
> 
> 
> 
> ```

Yeah that looks good to me and just about how I've fixed that in the past